.github/workflows/auto.yml

```diff
@@ -0,0 +1,44 @@
+name: Automatic CI for TFE Modules
+
+on:
+  pull_request:
+    branches:
+      - master
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  terraform_format:
+
+    name: Run terraform fmt
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Terraform
+      uses: hashicorp/setup-terraform@v1
+      with:
+        terraform_version: 0.14.7
+
+    - name: Format Public Install
+      run: terraform fmt -check -recursive -- ${{ github.workspace }}
```

Comment:
Let's include `-diff` so all parties can see what the issues are.

Is `--` necessary? I don't see it mentioned in the documentation.

Reply:
My bad on this, so used to writing up some bash that I accidentally added the `--`. I've added the `-diff` flag though.